### PR TITLE
Add wiki tags

### PR DESCRIPTION
**Pull request by meisyarahd**

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -23112,6 +23112,8 @@
     "count": 151,
     "tags": {
       "brand": "Marks & Spencer",
+      "brand:wikidata": "Q714491",
+      "brand:wikipedia": "en:Marks %26 Spencer",
```
Comment (good-praxis) [inline config/canonical.json]: I'd suggest changing "brand:wikipedia" to "en:Marks & Spencer"

**Comment by good-praxis** [comment]
this PR fixes #1604

**Comment by good-praxis** [comment]
@meisyarahd I've also noticed that your commits aren't connected to your Github account via e-mail, if you want your commits to properly count towards your account try configuring it like [this](https://help.github.com/enterprise/2.14/user/articles/setting-your-commit-email-address-in-git/)

**Comment by meisyarahd** [comment]
Done! both the change and the email setup :)

**Comment by good-praxis** [comment]
Thanks again 😁

**Pull request merged**